addon/services/cloudinary.js

```diff
@@ -181,5 +181,97 @@ export default Ember.Service.extend({
     }
 
     return `${protocol}${subDomain}.${domain}${pathname}`;
+  },
+
+
+  /** @property {String[]} concatenatedTransforms */
+  concatenatedTransforms: computed('config.CONCATENATED_TRANSFORMS', function() {
+    // Protection from Strings
+    const concatenatedTransforms = this.get('config.CONCATENATED_TRANSFORMS') || [];
+    return Array.isArray(concatenatedTransforms) ? concatenatedTransforms : [concatenatedTransforms];
+  }),
+  /** @property {String[]} defaultTransforms */
+  defaultTransforms: computed.readOnly('config.DEFAULT_TRANSFORMS'),
+  /** @property {String} defaultImageFormat */
+  defaultImageFormat: computed.readOnly('config.DEFAULT_IMAGE_FORMAT'),
```

Comment:
`defaultTransforms` and `defaultImageFormat` can be removed and directly inline `computeUrl` function